src/com/lilithsthrone/game/Game.java

```diff
@@ -3402,15 +3409,15 @@ public String addNPC(NPC npc, boolean isImported) throws Exception {
 			}catch(NumberFormatException ex) {
 				tallyCount = Integer.parseInt(npc.getId().split(",")[0]);
 			}
-			if(tallyCount>npcTally) {
-				npcTally = tallyCount;
+			if(tallyCount>npcTally.get()) {
+				npcTally.set(tallyCount);
```

Comment:
This is no longer atomic, since the get and set are separate. Perhaps something like this would work better:

```
npcTally.updateAndGet(x -> Math.max(x, tallyCount));
```

Reply:
You are correct, thanks for spotting that!